index.html

```diff
@@ -19,25 +19,25 @@ <h1>Welcome to Tic Tac Toe</h1>
       <table>
         <tr>
           <!-- the onclick calls a function called "handleClick" and passes itself to it -->
-          <td id="top-left" onclick="handleClick(this)" ></td>
-          <td id="top-middle" onclick="handleClick(this)" ></td>
-          <td id="top-right" onclick="handleClick(this)"></td>
+          <td id="0-0" onclick="handleClick(this)" ></td>
+          <td id="0-1" onclick="handleClick(this)" ></td>
+          <td id="0-2" onclick="handleClick(this)"></td>
         </tr>
         <tr>
           <!-- the onclick calls a function called "handleClick" and passes itself to it -->
-          <td id="middle-left" onclick="handleClick(this)" ></td>
-          <td id="middle-middle" onclick="handleClick(this)" ></td>
-          <td id="middle-right" onclick="handleClick(this)"></td>
+          <td id="1-0" onclick="handleClick(this)" ></td>
+          <td id="1-1" onclick="handleClick(this)" ></td>
+          <td id="1-2" onclick="handleClick(this)"></td>
         </tr>
         <tr>
           <!-- the onclick calls a function called "handleClick" and passes itself to it -->
-          <td id="bottom-left" onclick="handleClick(this)" ></td>
-          <td id="bottom-middle" onclick="handleClick(this)" ></td>
-          <td id="bottom-right" onclick="handleClick(this)"></td>
+          <td id="2-0" onclick="handleClick(this)" ></td>
+          <td id="2-1" onclick="handleClick(this)" ></td>
+          <td id="2-2" onclick="handleClick(this)"></td>
         </tr>
         <!-- @TODO create new rows and cells for the rest of your TTT board. -->
         <!-- @TODO give each new cell an id & an event listener to call the "handleClick" function -->
       </table>
     </div>
-  </body>
+  </body><script src="./scripts.js"></script>
```

Comment:
you'll want to move `<script src="./scripts.js"></script>` above `</body>`
ex: `<script src="./scripts.js"></script>`
`</body>`